app/Amortization/Amortization.php

```diff
@@ -83,17 +83,43 @@ public function create()
     public function preview()
     {
         $IsSuperLoan = Str::contains($this->order->businessType->slug, 'super');
+        $IsNoBs = Str::contains($this->order->businessType->slug, 'no_bs');
+        $IsNoBsRenewalLoan = Str::contains($this->order->businessType->slug, 'renewal');
+
+
         if ($IsSuperLoan && env('USE_SUPER_LOAN_CALC')) {
             return $this->getSuperLoaPaymentPlans();
+        } else if ($IsNoBs) {
+            if ($IsNoBsRenewalLoan) {
```

Comment:
What are we doing here?

Reply:
checks if it is nobs first, then check if renewal or not